compiler/rustc_hir_typeck/src/check.rs

```diff
@@ -122,7 +122,14 @@ pub(super) fn check_fn<'a, 'tcx>(
         hir::FnRetTy::DefaultReturn(_) => body.value.span,
         hir::FnRetTy::Return(ty) => ty.span,
     };
-    fcx.require_type_is_sized(declared_ret_ty, return_or_body_span, traits::SizedReturnType);
+
+    if !declared_ret_ty.is_scalable_simd() {
```

Comment:
This needs a TODO to make sure it's not accidentally committed.

Reply:
Yeah, but it specifically says:

> or now just remove the check for testing purposes.

So it either needs a TODO to remove this once "testing purposes" are over, or the comment should be reworked to explain why it's correct that makes it seem less temporary.